format/format_test.go

```diff
@@ -190,6 +190,38 @@ var _ = Describe("Format", func() {
 				Expect(MessageWithDiff(stringWithB, "to equal", stringWithZ)).Should(Equal(expectedFullFailureDiff))
 			})
 		})
+
+		Context("With alternate diff lengths", func() {
+			initialValue := TruncateThreshold // 5 by default
+			BeforeEach(func() {
+				TruncateThreshold = 10000
+			})
+
+			AfterEach(func() {
+				TruncateThreshold = initialValue
+			})
+			It("should show the full diff", func() {
+				stringWithB := "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+				stringWithZ := "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaazaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+
+				Expect(MessageWithDiff(stringWithB, "to equal", stringWithZ)).Should(Equal(expectedFullFailureDiff))
+			})
+		})
+
+		Context("With alternate diff lengths", func() {
+			It("long strings that differ only in length", func() {
```

Comment:
Ideally BDD style tests should read as a sentence. For example the test above reads `it should show the full diff`, and this one could read something like `it shows more characters around a line length mismatch` - test names should describe the *behaviour* being tested rather than the implementation of the test.